Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -610,6 +611,13 @@ public extension Tensor {
 }
 
 public extension Tensor where Scalar: Numeric {
+    /// Returns a `Tensor` by clipping tensor values to a specified min and max.
+    @inlinable
+    @differentiable(wrt: self where Scalar: TensorFlowFloatingPoint)
+    func clipped(min minimum: Tensor, max maximum: Tensor) -> Tensor {
```

Comment:
Line 580 indicates that this block of code should be about broadcasting. However, `clipped(min:max:)` is not about broadcasting. It should be moved to an extension in an appropriate place.